xdsl/parser.py

```diff
@@ -757,6 +760,63 @@ def parse_list_of(self,
 
         return items
 
+    def parse_optional_integer(self,
```

Comment:
Could you reuse this for `_parse_int_or_question`? :thinking:

Reply:
Good point! Yes